scripts/submit_gaussian.bash

```diff
@@ -46,7 +51,7 @@ check_for_jobs()
 	echo "Output, checkpoint and formatted checkpoint files will be placed under:"
 	echo "$OUTPUT_DIR/"
 	echo "Use the following command to check on your job queue:"
-	echo '	squeue -u $USER'
+	echo '	squeue -u' $USER
```

Comment:
Is this branch up to date with master? This change is already made on the master branch.

Reply:
I made the branch before that change.

Reply:
ya, if you merge master into this branch then this change shouldn't appear.